Assets/_BForBoss/_Entities/Scripts/LifeCycleManager.cs

```diff
@@ -9,6 +9,7 @@ public class LifeCycleManager : MonoBehaviour
         public Action<int> OnLivingEntityEliminated;
```

Comment:
General note for this class, since we now have `PlayerLifeBehaviour` which inherits from `LifeBehaviour`. It seems weird the `LifeBehaviorManager` handles the tallying (or event the initialization itself) of Enemies that have been killed. That sort of logic should be handled by some sort of General Enemy Manager class. Which should also handle initializing enemies integrated in the scene and enemies generated from the spawner.

If that makes sense to you @m4rtint , I can go ahead and create a ticket for that (i.e. clean up `LifeCycleManager` and create `EnemyManager`)? It can be handled after #201 gets merged

Reply:
Yup okay makes sense